src/view/pages/FarmerSearch/index.tsx

```diff
@@ -148,7 +89,7 @@ class FarmerSearch extends React.Component<PropsType, OwnStateType> {
   /** Function to render the individual list items */
   private renderItem(info: FarmerType) {
     return (
-      <ListItem key={info.id} onPress={this.itemClicked}>
+      <ListItem key={info.uuid} onPress={this.itemClicked}>
```

Comment:
I know this isn't your page and wasn't your mistake, but can you change this to `farmer` instead of `info`? It's more specific as to what it actually is. I think I might have made the same change in the branch you just reviewed tbh so if I did then disregard